src/main/js/field/EndpointSelectField.js

```diff
@@ -74,6 +74,7 @@ class EndpointSelectField extends Component {
 }
 
 EndpointSelectField.propTypes = {
+    id: PropTypes.string,
```

Comment:
Is this property used?

Reply:
Yes the DynamicSelectInput has `{..this.props}` Which will use the id of the field and set it on the DynamicSelectField which does use the id property when rendering.